README.CentOS7.md

```diff
@@ -78,7 +78,8 @@ Then follow instructions to create a password for the root user.
 
 ## NodeJS
 ```bash
-sudo yum install nodejs
+curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
+sudo yum install -y nodejs
```

Comment:
```suggestion

```
This line seam to be not needed.  Script downloaded and ran at line 81, install the RPM.

Reply:
@PapillonMcGill oops, thank you!